src/main/sensors/gyro.h

```diff
@@ -61,6 +61,9 @@ typedef struct gyroConfig_s {
     uint16_t gyro_soft_notch_cutoff_1;
     uint16_t gyro_soft_notch_hz_2;
     uint16_t gyro_soft_notch_cutoff_2;
+#if !defined(BEEBRAIN)
```

Comment:
I'd remove this - it's hardcoded at the moment and we can add this if we decide to make it user-definable.